src/components/Campaign.tsx

```diff
@@ -28,6 +31,28 @@ export const Campaign = () => {
   const campaignContentQuery = useGetCampaignContent(idCampaign);
   const campaignContentMutation = useUpdateCampaignContent();
 
+  const [showModal, hideModal] = useModal(() => {
+    const { campaignName, htmlContent, type, previewImage, design } =
+      campaignContentQuery.data!;
+
+    // TODO: Improvement the component when return html content
+    return type === "unlayer" ? (
+      <SaveAsTemplateModal
+        content={{
+          templateName: campaignName,
+          htmlContent,
+          type,
+          design,
+          previewImage,
+          isPublic: false,
+        }}
```

Comment:
In my opinion, the property `content` should receive an `UnlayerContent`, not more and not less.

Reply:
Ok, but we need pass the template name in props.

Reply:
If yo want to use the campaign name as _default template name_ you can pass it.

But I am not sure if it makes sense. How it is working now in MSEditor?

Reply:
![24-01-2023_16-17-07](https://user-images.githubusercontent.com/6733401/214418363-b4181809-3dc4-4fb1-8bdb-b4331afbb12b.gif)

Reply:
So, it is using the campaign name, right?

Reply:
Yes, that use the campaign name.